procedures/MT/rotator/rotator_verification.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>I imagine this is obvious to you but not to me :-)</p><p>What is the "correct" version of the CSC? Clearly it's not been deployed for a reason?</p><p>Is this in cycle 27 or something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Ah ok. This makes sense, consider:</p><p>"The following cells work around a rotator CSC telemetry problem with the version on the summit (2022-09-29). This is fixed and will no longer be required when the new version is deployed.  </p><p> </p>

Reply:
Updated in the commit: [3a3d19c](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/3a3d19ca25f47a8afd6d0e2d48a2c4c3cc33224d). Thanks!